scripts/install-atom.ps1

```diff
@@ -4,3 +4,7 @@ $env:PATH="$env:PATH;$env:LOCALAPPDATA\atom\bin"
 apm install language-powershell
 apm install language-batch
 apm install language-docker
+apm install franken-berry-ui
+apm install franken-berry-syntax
+apm install file-icons
+apm install platformio-ide-terminal
```

Comment:
Great idea, so how about I then create an additional .ps1 file call it something like **atom-add-ons.ps1** and include various `apm install` commands for items that are not part of the repo?

Makes sense on the themes and not installing them as they are in fact a personal preference.  I'll remove those.